Guess game/Program.cs

```diff
@@ -10,282 +10,98 @@ namespace Guess_game
 {
     class Program
     {
+        private static void AnswerMessage(bool correct)
+        {
+            Console.WriteLine(correct ? "Nice you get it " : "You did not get it ");
+
+            Console.Write("Press Enter to continue ");
+            Console.ReadLine();
+        }
+
+
         private static Dictionary<string, string> Riddles = new Dictionary<string, string>
         {
-            // { key, value }
-            {"zalia zole bet ne zole su uodega bet nepele?" , "agurkas"},
-            { "be ranku be koju duris atidaro", "vejas"},
+            {"zalia zole bet ne zole su uodega bet ne pele?" , "agurkas"},
+            { "be ranku be koju duris atidaro?", "vejas"},
             { "I pakalne letai i kalna greitai?", "snarglys"}
         };
-        // foreach 
 
         static void Main(string[] args)
         {
-            //var answers = new Dictionary<string, int>();
-            
+            int Result = 0;
 
-            //var guess = string.Empty;
-            //// for / foreach
-            //foreach (var riddle in Riddles)
-            //{
-            //    answers.Add(riddle.Key, 0);
+            Intruduction();
 
-            //    while (guess != riddle.Value)
-            //    {
-            //        Console.Write("Enter your guess ");
-            //        guess = Console.ReadLine();
+            Result += RiddleGame(0);
```

Comment:
why do you pass 0 to this method ? (as i understand parameters to this method is not needed, remove it)

Reply:
`int result = 0;` line is not needed as you can write just: `int result = RiddleGame();`